deploy/zfs-operator.yaml

```diff
@@ -512,6 +512,9 @@ rules:
   - apiGroups: [""]
     resources: ["secrets"]
     verbs: ["get", "list"]
+  - apiGroups: [""]
+    resources: ["namespaces"]
```

Comment:
should access to node be provided?

Reply:
that is already there as a part of openebs-zfs-attacher-role.